app/src/main/java/io/mochahub/powermeter/data/ExerciseDao.kt

```diff
@@ -12,6 +12,15 @@ interface ExerciseDao {
     @Query("SELECT * FROM exercises ORDER BY createdAt ASC")
     fun getAll(): Flow<List<ExerciseEntity>>
 
+    @Query("SELECT * FROM exercises ORDER BY createdAt ASC")
+    fun getAllAsList(): List<ExerciseEntity>
+
+    @Query("SELECT * FROM exercises where name=:exerciseName")
+    fun findByName(exerciseName: String): ExerciseEntity
+
+    @Query("SELECT * FROM exercises where id=:exerciseID")
+    fun findByID(exerciseID: String): ExerciseEntity
```

Comment:
`suspend fun` for all these to avoid running them on the main thread

Reply:
Implemented here https://github.com/power-meter/android-client/pull/32/commits/71310f83b5b8e194501a55222c253af923395220